src/main/webapp/i18n/de/featureOverview.json

```
@@ -39,7 +39,7 @@
                 "sessionMonitoring": {
                     "title": "Überwachung der Klausur",
                     "shortDescription": "Überwache die Klausursitzung, um ungewöhnliche Aktivitäten zu identifizieren.",
-                    "descriptionTextOne": "Ohne eine Aufsicht, kann es sich so anfühlen, als könntest du deine Studierenden nicht genau beobachten. Um dem entgegenzuwirken, beinhaltet Artemis eine Überwachung der Klausuren. Für alle Studierenden, die an der Klausur teilnehmen, generiert Artemis eine Klausursitzung. Die Klausursitzung enthält neben anderen Feldern, wann und wo die Klausur gestartet wurde, die öffentliche IP-Adresse und einen einzigartigen Fingerabdruck, der auf dem Browser und dem Betriebssystem basiert. Zusätzlich generiert die Klausur eine eindeutige Kennung, die mit jeder Anfrage gesendet wird. Studierende haben die Möglichkeit, während der Klausur zwischen den Computern zu wechseln und auch offline weiterzuarbeiten. Als Ausbilder haben Sie die Möglichkeit, alle Sitzungen und das Verhalten der Studierenden zu analysieren und verdächtig erscheinende Fälle zu markieren."
+                    "descriptionTextOne": "Ohne eine Aufsicht, kann es sich so anfühlen, als könntest du deine Studierenden nicht genau beobachten. Um dem entgegenzuwirken, beinhaltet Artemis eine Überwachung der Klausuren. Für alle Studierenden, die an der Klausur teilnehmen, generiert Artemis eine Klausursitzung. Die Klausursitzung enthält neben anderen Feldern, wann und wo die Klausur gestartet wurde, die öffentliche IP-Adresse und einen einzigartigen Fingerabdruck, der auf dem Browser und dem Betriebssystem basiert. Zusätzlich generiert die Klausur eine eindeutige Kennung, die mit jeder Anfrage gesendet wird. Studierende haben die Möglichkeit, während der Klausur zwischen den Computern zu wechseln und auch offline weiterzuarbeiten. Als Ausbilder hast Du die Möglichkeit, alle Sitzungen und das Verhalten der Studierenden zu analysieren und verdächtig erscheinende Fälle zu markieren."
```

imo this sounds a bit strange:
```suggestion
                    "descriptionTextOne": "Ohne eine Aufsicht kann es sich so anfühlen, als würdest du deine Studierenden nicht genau beobachten. Um dem entgegenzuwirken, beinhaltet Artemis eine Überwachung der Klausuren. Für alle Teilnehmenden generiert Artemis eine Klausursitzung. Diese Klausursitzung enthält unter anderem, wann und wo die Klausur gestartet wurde, die öffentliche IP-Adresse und einen einzigartigen Fingerabdruck, der auf dem Browser und dem Betriebssystem basiert. Zusätzlich generiert die Klausur eine eindeutige Kennung, die mit jeder Anfrage gesendet wird. Studierende haben die Möglichkeit, während der Klausur zwischen den Computern zu wechseln und auch offline weiterzuarbeiten. Als Ausbilder hast Du die Möglichkeit, alle Sitzungen und das Verhalten der Studierenden zu analysieren und verdächtig erscheinende Fälle zu markieren."
```